src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -87,15 +94,38 @@ const ProfilePictureInput: React.FunctionComponent<ProfilePictureInputProps> = (
     setValue("pictureUrl", pictureUrlRef, true);
   };
 
-  const uploadDefaultAvatar = async (avatar: string) => {
-    setValue("pictureUrl", avatar, true);
-  };
+  const uploadDefaultAvatar = useCallback(
+    async (avatar: string) => {
+      setValue("pictureUrl", avatar, true);
+    },
+    [setValue]
+  );
+
+  const isLoading =
+    (isLoadingSovereignVenueId || isLoadingCustomAvatars) &&
+    (customAvatars !== undefined || error !== undefined);
 
-  const isLoadingAvatars = customAvatars.loading;
+  const defaultAvatars = customAvatars?.length
+    ? customAvatars
+    : DEFAULT_AVATARS;
 
-  const defaultAvatars = customAvatars?.value?.length
-    ? customAvatars.value
-    : sparkleAvatars;
+  const avatarImages = useMemo(() => {
+    return defaultAvatars.map((avatar, index) => {
+      return (
+        <div
+          key={`${avatar}-${index}`}
+          className="profile-picture-preview-container"
+          onClick={() => uploadDefaultAvatar(avatar)}
+        >
+          <img
+            src={`${avatar}`}
```

Comment:
```suggestion
            src={avatar}
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1409/commits/041df52e62699a3a610407e6d105970072a53ce2